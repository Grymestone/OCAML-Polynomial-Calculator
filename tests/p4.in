(2a + 39a^2)^4
