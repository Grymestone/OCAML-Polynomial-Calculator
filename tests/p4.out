(((2*a)+(39*(a)^2)))^4
474552x^7 + 2313441x^8 + 36504x^6 + 1248x^5 + 16x^4